spec/views/users/show.html.erb_spec.rb

```diff
@@ -1,5 +1,16 @@
 require 'rails_helper'
 
-RSpec.describe 'users/show.html.erb', type: :view do
-  #  pending "add some examples to (or delete) #{__FILE__}"
+RSpec.feature 'User Show Page' do
+  scenario 'Viewing user details' do
+    # Create a test user directly in the test database
+    user = User.create(name: 'Test User', bio: 'Test Bio')
+
```

Comment:
- [ ] It would be better if we implement the required test, for example, if we want to test for the user's first 3 posts, we can test the method `recent_posts` in the testing. Here is a [link](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/projects/intergration_spec_and_n_1_project.md) to where we can find all our required test
![Screenshot from 2023-09-20 16-19-46](https://github.com/Gardimy/BlogApp/assets/46388113/0df86218-14ae-4376-b220-f8dd6f34a82c)

Reply:
- [ ] @Team as previously requested kindly consider adding tests that ascertain you can see the user's first 3 posts. it's a project requirement
![image](https://github.com/Gardimy/BlogApp/assets/61408860/bbf28dd0-e397-4f1f-b033-82922799bbdb)

- [ ] [optional] I also noted you are missing a test that shows a user is redirected to the post's show page when the user clicks on a user's post.